setup.sh

```diff
@@ -70,6 +70,7 @@ pssh --inline \
     --host "$MASTERS $SLAVES" \
     --user root \
     --extra-args "-t -t $SSH_OPTS" \
+    --timeout 300 \
```

Comment:
It [doesn't look like](https://github.com/mesos/spark-ec2/blob/9f67e96e4763a03931c0b1ab3f26b6faa8f6266e/setup.sh#L30) we had anything in previously to set the timeout apart from a connection timeout (which is preserved with the pssh call). So the SSH timeout would just depend on the OS's default `TCPKeepalive` setting, I believe.